Building.md

```diff
@@ -0,0 +1,43 @@
+This is the general getting started guide for FullScreenShenanigans.
+
+### Table of Contents
+
+1. [Obtaining the Source Code](#obtaining-the-source-code)
+2. [Build Process](#build-process)
+3. [Using the Console](#using-the-console)
+
+## Obtaining the Source Code
+
+In order to play FullScreenShenanigans, you first need to obtain the source code. There are two ways to do this:
+
+A. Clone the repository:
+1. Copy the clone URL (URL located to the left of the Download Zip button).
+2. Open Git Bash and change the current working directory to the location where you want the cloned directory to be made. 
+3. Type git clone and paste the URL you copied then press enter. You now have your own copy of FullScreenShenanigans! See [Build Process](#build-process) to see how to finish setup and play the game. 
+
+B. Download the latest release .zip of this project, extract that onto your computer, and open index.html in a browser (preferably Google Chrome). That's it!
+
+
+## Build Process
+
+FullScreenShenanigans uses [Grunt](http://gruntjs.com/) to automate building, which requires [Node.js](http://node.js.org). 
+
+Go to the root repository folder (containing the Getting Started guide) and run the following commands:
+
+    npm install
+    grunt
+    
+You may also have to manually install grunt first:
+
+    npm install -g grunt grunt-cli
+
+All of the files in Source, will be compiled into a new Distribution folder in the root.
+
+* FullScreenShenanigans.d.ts and FullScreenShenanigans.ts are merged into a FullScreenShenanigans.ts, and all .ts files under References are copied as well. A .js file with a .min.js and .min.js.map will be generated as well.
```

Comment:
For this first bullet, can you be more clear? It's a vague summary of information that would be good to have details on.